src/main/kotlin/org/jitsi/nlj/stats/EndpointConnectionStats.kt

```diff
@@ -113,7 +113,12 @@ class EndpointConnectionStats(
             rtt = (Duration.between(srSentTime, receivedTime) - remoteProcessingDelay).toDoubleMillis()
             if (rtt > Duration.ofSeconds(7).toMillis()) {
```

Comment:
Should we change this value or change the log level of this as well? As of now it's just noise I think.

Reply:
Sorry, by lower I meant debug

Reply:
Do you think the negative values should still be warning for now (at least for a cycle through meet)?

Reply:
Sure...but did we see something that makes us think we're seeing negative RTTs?

Reply:
Yes, as I mentioned in the original message.  Boris's review of the logs showed that we saw occasional `java.lang.IllegalArgumentException: responseTimeMs` messages from `AimdRateControl`, and inspecting the code, that can only happen in rtt is -100 ms or less.

Reply:
👍 Sorry, forgot about that.